Kernel/Memory/MemoryManager.cpp

```diff
@@ -190,7 +190,7 @@ UNMAP_AFTER_INIT void MemoryManager::register_reserved_ranges()
     m_reserved_memory_ranges.append(ContiguousReservedMemoryRange { range.start, m_physical_memory_ranges.last().start.get() + m_physical_memory_ranges.last().length - range.start.get() });
 }
 
-bool MemoryManager::is_allowed_to_mmap_to_userspace(PhysicalAddress start_address, VirtualRange const& range) const
+bool MemoryManager::is_allowed_to_mmap_physical_memory_to_userspace(PhysicalAddress start_address, VirtualRange const& range) const
 {
     VERIFY(!m_reserved_memory_ranges.is_empty());
```

Comment:
I know this is not part of this PR, but what's the point of this VERIFY? the rest of the method doesn't assume there is at least one reserved memory range, and I don't see why that is guaranteed anyways.

Reply:
I saw it yesterday and I could not agree more on that...  probably should be removed later :)

Reply:
Fixed in #11697.